modules/speech-to-text-api/src/main/java/org/opencastproject/speechtotext/api/SpeechToTextEngine.java

```diff
@@ -36,13 +37,14 @@ public interface SpeechToTextEngine {
   /**
    * Generates the subtitles file.
    *
-   * @param mediaFile The media package containing the audio track.
+   * @param mediaFile          The media package containing the audio track.
    * @param preparedOutputFile The prepared output file where the subtitle's data should be saved.
-   * @param language The language of the audio track.
+   * @param language           The language of the audio track.
+   * @param translate          If the subtitles should be translated into english
    * @return The generated subtitles file.
```

Comment:
Since you changed the return type, you should change the return description as well